tophat/jenkins-stats-publisher-plugin
# feat: Add support for getting api key from envvars

## Description

This allows us to use the Jenkins environment variables to set an API key, no more hard coding, ever!

## DevQA
- Go to `Manage Jenkins` then `Configure system`
- Set the API key as an environment variable (see under `Global properties`)
![image](https://user-images.githubusercontent.com/6210361/56664805-cba0d500-6676-11e9-8a67-e7204f5a73f2.png)
- Run a build and verify that the printed status code is `202`, signalling that the payload was accepted by DD.
- A 403 is what we would see with a missing API key, which you can also test. The system should be resilient to the absence of a key and should just 403 when sending to DD.
